stories/Button-stories.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * Echoes React
+ * Copyright (C) 2023-2024 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+/* eslint-disable no-console */
+import styled from '@emotion/styled';
+import type { Meta, StoryObj } from '@storybook/react';
+import { Button, ButtonSize, ButtonVariety, IconCalendar, IconChevronDown } from '../src';
+
+const meta: Meta<typeof Button> = {
+  component: Button,
+  title: 'Echoes/Button',
+  argTypes: {
+    size: { control: { type: 'select' }, options: Object.values(ButtonSize) },
+    variety: { control: { type: 'select' }, options: Object.values(ButtonVariety) },
+  },
+};
+
+export default meta;
+
+type Story = StoryObj<typeof Button>;
+
+export const Default: Story = {
```

Comment:
maybe you can use the [BasicWrapper](https://github.com/SonarSource/echoes-react/blob/main/stories/DropdownMenu-stories.tsx#L117) we use in other stories so that the component isn't on the top-left?